manifests/nfd-daemonset-master-topology-updater.yaml.template

```diff
@@ -0,0 +1,146 @@
+# This template contains an example of running nfd-master and nfd-worker in the
+# same pod.
+#
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: node-feature-discovery # NFD namespace
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nfd-master
+  namespace: node-feature-discovery
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nfd-master
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    # when using command line flag --resource-labels to create extended resources
+    # you will need to uncomment "- nodes/status"
+    # - nodes/status
+    verbs:
+      - get
+      - patch
+      - update
+      # List only needed for --prune
+      - list
+  - apiGroups:
```

Comment:
I tested manifest created from this template and it is not creating CRDs as per the previous manifest template. I think the reason is that we are missing "rbac.authorization.k8s.io" apigroup (which was in the previous manifest). We would have to make changes to all the manifest with nfd-master that are trying to access (create/update/patch) CRDs